client/src/app.js

```diff
@@ -51,7 +51,16 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
     , last_txids: parseHashes(loc.query.txids)
     , est_chain_seen_count: +loc.query.c || 0
     }))
-  , goAssetList$ = !process.env.IS_ELEMENTS || !process.env.ASSET_MAP_URL ? O.empty() : route('/assets')
+  //, goAssetList$ = !process.env.IS_ELEMENTS || !process.env.ASSET_MAP_URL ? O.empty() : route('/assets')
+  , goAssetList$ = !process.env.ASSETLIST_URL ? O.empty() : route(process.env.ASSETLIST_URL).map(loc => {
```

Comment:
This doesn't need to use `process.env.ASSETLIST_URL`, you can check for `IS_ELEMENTS` and use a literal string for `route()`.